server/neptune/terraform/subcommand.go

```diff
@@ -85,6 +86,12 @@ func (c *SubCommand) WithArgs(args ...Argument) *SubCommand {
 	return c
 }
 
+// WithInput adds a single command input
+func (c *SubCommand) WithInput(input string) *SubCommand {
```

Comment:
this is fine, but this would be pretty cool to have as an io reader to basically allow piping of different commands together (obviously overkill here)